<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>virtuaClass</class>
 <widget class="QMainWindow" name="VirtualRigClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>virtua</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>717</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Tool</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="SkelVisibleCheckBox">
       <property name="toolTip">
        <string>Skeleton (Joints)</string>
       </property>
       <property name="text">
        <string>Show Skeleton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bvhVisibleCheckBox">
       <property name="toolTip">
        <string>BVH Skeleton</string>
       </property>
       <property name="text">
        <string>Show BVH Skeleton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ErrorVisibleCheckBox">
       <property name="toolTip">
        <string>Error(lose volume)</string>
       </property>
       <property name="text">
        <string>Show Error</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="WireVisibleCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Show WireFrame</string>
       </property>
       <property name="text">
        <string> Wireframe </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Stiffnesslabel_2">
       <property name="text">
        <string>Stretch Stiffness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Stiffness_str">
       <property name="toolTip">
        <string>Fat Tissue Siffness</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="sliderPosition">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="buttonPlay">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Play animation (loop)</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonPause">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Pause</string>
          </property>
          <property name="text">
           <string>||</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonOneStep">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Next frame</string>
          </property>
          <property name="text">
           <string>&gt;|</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Tpose">
       <property name="toolTip">
        <string>T-Pose( init)</string>
       </property>
       <property name="text">
        <string>T-Pose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ObjButton">
       <property name="toolTip">
        <string>Linear Blend Skinning</string>
       </property>
       <property name="text">
        <string>Save obj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Loadmeshlist">
       <property name="text">
        <string>Load Mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modelComboBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pick a rigged, mesh to animate.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="maxVisibleItems">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Loadbvhlist">
       <property name="text">
        <string>Load BVH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="BVHComboBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pick a rigged, mesh to animate.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="maxVisibleItems">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="bvhListView">
       <property name="toolTip">
        <string>Load new BVH</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="virtua.qrc"/>
 </resources>
 <connections/>
</ui>
